blocktopmenu.php

```diff
@@ -52,7 +52,7 @@ public function __construct()
     {
         $this->name = 'blocktopmenu';
         $this->tab = 'front_office_features';
-        $this->version = '2.2.2';
+        $this->version = '2.3';
```

Comment:
Hi,

The merger will adapt the file when he will create a new master version if necessary.

Regards